packages/core/src/external/fhir/patient/index.ts

```diff
@@ -19,7 +20,7 @@ export const genderMapping: { [k in GenderAtBirth]: "female" | "male" } = {
 export const toFHIR = (patient: Pick<Patient, "id" | "data">): FHIRPatient => {
   return {
     resourceType: "Patient",
-    id: patient.id,
+    id: uuidv4(),
```

Comment:
gen a random id for now. Not clear what value if any this id serves (the only important id is the externalGatewayPatient.id

Reply:
there was some discussion between uuidv4 vs uuidv7, where we prefer the latter - https://github.com/metriport/metriport/pull/1168#discussion_r1396067720